pkg/metainfo/kvmetainfo/buckets.go

```diff
@@ -18,7 +18,10 @@ func (db *Project) CreateBucket(ctx context.Context, bucketName string, info *st
 		return storj.Bucket{}, storj.ErrNoBucket.New("")
 	}
 
-	meta, err := db.buckets.Put(ctx, bucketName, getPathCipher(info))
+	newBucketMeta := buckets.Meta{
+		PathEncryptionType: getPathCipher(info),
```

Comment:
We need to fill all other fields of `buckets.Meta`.

Also, we need to provide default values if `info == nil`.

Reply:
Well spotted. Thanks, will fix